base_comment_template/tests/test_base_comment_template.py

```diff
@@ -37,99 +37,84 @@ def setUpClass(cls):
                 "position": "after_lines",
                 "text": "Text after lines",
                 "model_ids": [(6, 0, cls.user_obj.ids)],
-                "priority": 6,
+                "sequence": 6,
                 "company_id": cls.company.id,
             }
         )
+        cls.user.partner_id.base_comment_template_ids = [
+            (4, cls.before_template_id.id),
+            (4, cls.after_template_id.id),
+        ]
 
     @classmethod
     def tearDownClass(cls):
         teardown_test_model(cls.env, ResUsers)
         super(TestCommentTemplate, cls).tearDownClass()
 
+    def test_template_name_get(self):
+        self.assertEqual(
+            self.before_template_id.display_name, "before_lines (before_lines)",
```

Comment:
The `name_get` should return a proper human readable text, not `before_lines`. Use the natural value (with `_fields[field_name]._get_selection` or similar method name).